### PR TITLE
update gcp docs to make it clear how to limit infra hosts

### PR DESCRIPTION
**Pull request by goeln**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by rtrieu** [comment]
Created DOCS-7479 for a docs editorial review.

File: content/en/cloud_cost_management/google_cloud.md [hunk 1]
```diff
@@ -128,10 +137,10 @@ You can visualize your ingested data using the following cost types:
 | Cost Type | Description |
 | ----------| ----------------------------------|
 | `gcp.cost.amortized` | Total cost of resources allocated at the time of usage over an interval. Costs include promotion credits as well as committed usage discount credits. |
-| `gcp.cost.amortized.shared.resources.allocated` | All of your GCP amortized costs, with additional breakdowns and insights for container workloads. Requires [container cost allocation][15].|
+| `gcp.cost.amortized.shared.resources.allocated` | All of your GCP amortized costs, with additional breakdowns and insights for container workloads. Requires [container cost allocation][14].|
 
 ### Container allocation
-**Container allocation** metrics contain all of the same costs as the GCP metrics, but with additional breakdowns and insights for container workloads. See [container cost allocation][15] for more details.
+**Container allocation** metrics contain all of the same costs as the GCP metrics, but with additional breakdowns and insights for container workloads. See [container cost allocation][14] for more details.
```
Comment (buraizu) [inline content/en/cloud_cost_management/google_cloud.md:143]:
```suggestion
**Container allocation** metrics contain all of the same costs as the Google Cloud Platform metrics, but with additional breakdowns and insights for container workloads. See [Container Cost Allocation][14] for more details.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 2]
```diff
@@ -20,17 +20,23 @@ further_reading:
 
 ## Overview
 
-To use Google Cloud Cost Management in Datadog, you must configure the [Google Cloud Platform Integration][12] and set up the [detailed usage cost][14] billing export for your desired billing account.
-Additionally, the Google Cloud Platform Datadog Integration Service Account must have the [necessary permissions][13] configured to interact with Google Cloud Storage and BigQuery.
+To use Google Cloud Cost Management in Datadog, you must:
```
Comment (buraizu) [inline content/en/cloud_cost_management/google_cloud.md:23]:
```suggestion
To use Google Cloud Cost Management in Datadog, follow these steps: 
```

**Comment by goeln** [comment]
thanks! lgtm to merge :)

**Review by buraizu — APPROVED** [review]

**Pull request merged**